equinix-openapi-metal/src/main/java/com/equinix/openapi/metal/v1/model/ActivateHardwareReservationRequest.java

```diff
@@ -14,13 +14,13 @@
 package com.equinix.openapi.metal.v1.model;
 
 import java.util.Objects;
-import java.util.Arrays;
 import com.google.gson.TypeAdapter;
 import com.google.gson.annotations.JsonAdapter;
 import com.google.gson.annotations.SerializedName;
 import com.google.gson.stream.JsonReader;
 import com.google.gson.stream.JsonWriter;
 import java.io.IOException;
+import java.util.Arrays;
```

Comment:
I mean, moving this import is just unnecessary (and probably stylistically worse), but OK.